src/main/java/cn/nukkit/item/ItemDye.java

```diff
@@ -74,13 +74,6 @@ public ItemDye(Integer meta, int amount) {
     protected ItemDye(int id, Integer meta, int count, String name) {
         super(id, meta, count, name);
     }
-
-    @Since("1.4.0.0-PN")
-    @PowerNukkitOnly
-    @Override
-    public boolean isFertilizer() {
-        return getId() == DYE && getDyeColor().equals(DyeColor.WHITE);
-    }
```

Comment:
mark as deprecated and keep the method returning false to avoid plugin compatibility issues

Reply:
Actually, `isFertilizer` is a PowerNukkit API (or atleast it is on HEAD), with the default being returning `false`.
Meaning that you can just remove this function and @LoboMetalurgico was right.